core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3396,10 +3396,6 @@ private boolean syncMetadataInternal(RpcContext rpcContext, LockedInodePath inod
 
     // Update metadata for all the mount points
     for (String mountPoint : pathsToLoad) {
-      if (Thread.currentThread().isInterrupted()) {
```

Comment:
can we keep both if conditions? if we cancel before syncing the first path, we would sync the first path with this change.

Reply:
done